lib/application.js

```diff
@@ -153,6 +153,20 @@ Application.prototype.createClient = function () {
       args.push(arg)
     })
 
+    // Since ChromeDriver v2.41, ChromeDriver with CEF will only connect
```

Comment:
Electron doesn't use CEF :) Do you mean this is broken here in a similar way it's broken in CEF?

Reply:
Ah yes, it was not precise, but yes, I guess it's similar behavior.